src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -79,6 +77,13 @@ public Case saveCaseSample(CreateCaseSample createCaseSample) {
     caze.setCeActualResponses(0);
     caze.setHandDelivery(isTreatmentCodeDirectDelivered(createCaseSample.getTreatmentCode()));
 
+    if (caze.getCaseType().equals("CE")) {
```

Comment:
Not particularly keen on building this logic specifically for CE's, I thought our general policy was to avoid conditioning on things like case type unless the features really require it? (to clarify though, I do definitely like the idea of storing sparse columns like this into a metadata json column, I just don't think we should build it to be specific to any particular case type).

Reply:
I agree.  I thought with 'secure estab' for example that we send true or false, so this could be stored for any case?